frontend/src/Onboarding/GenericOnboarding.tsx

```diff
@@ -57,7 +58,7 @@ const theme = createMuiTheme({
   },
 });
 
-const steps = ["", ""];
+const steps = ["", "", ""];
```

Comment:
What is the purpose of steps? Why does it need to be initialized to an array of empty strings?

Reply:
I just use it so I can use .map to create the Steps. It doesn't really need to be an empty array but it's just ease of use